### PR TITLE
Bug fix about dataloader and inlier beta

### PR DESCRIPTION
**Pull request by qiyan98**
Hi everyone!

I want to report and fix some minor (but important) bugs here.

[1](https://github.com/vislearn/esac/blob/master/code/ref_expert.py#L46). `from dataset import RoomDataset` should be `from room_dataset import RoomDataset`

[2](https://github.com/vislearn/esac/blob/master/code/train_esac.py#L166). it should be `opt.inlierbeta` rather than`opt.inlieralpha` here

[3](https://github.com/vislearn/esac/blob/master/code/test_esac.py#L203). similar to the last one, it should be `opt.inlierbeta` here.

I am not very proficient with git, and I hope I am doing the correct things here. :)

### CODE REVIEW

**Comment by qiyan98** [comment]
By the way, before those bugs were found, I have done quite a lot of training using `train_esac.py`. I was just wondering if the wrong use of `inlierbeta` parameter is going to have a major impact on the final result or not? As far as I understand, the softness of hypotheses was much less soft than it should have been (100 vs 0.5). Should I expect a significant performance improvement with correct beta parameters?

**Comment by ebrach** [comment]
Great catch, and thank you very much for creating a pull request. The incorrect use of inlierbeta would make the soft inlier count practically a hard inlier count. I would expect that during test time, that would not change much. At training time, you would get no gradients of the scoring function. But in my experience gradients of the inlier count tend to be small, and they are usually dominated by the gradients of PnP. So, I would expect you to get similar results with either choice of inlierbeta, but please let me know if you find out more on this :-)
